tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "forceConsistentCasingInFileNames": true,
     "noFallthroughCasesInSwitch": true,
     "noImplicitReturns": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Why is this needed?

Reply:
TS won't compile with unused vars in the controller. (res, next, etc).

Reply:
Let's remove unused var instead of disabling here